src/commands/case_management.py

```diff
@@ -387,10 +395,8 @@ async def cmd_case_management_inject(ctx: Context):
             for keyword in ctx.words_eol[2].split():
```

Comment:
[Question] Why are we still using this loop at all when we have `platform` in the pyparsing object now?

Reply:
hold over from when i didn't have platform detection in the pyparsing object. will correct.